app/model/entity/Assignment.php

```diff
@@ -407,13 +424,20 @@ function ($key, RuntimeEnvironment $env) {
             })
         ],
         "runtimeEnvironments" => [
-          "upToDate" => $this->getRuntimeEnvironments()->count() === $this->getExercise()->getRuntimeEnvironments()->count()
-            && $this->getRuntimeEnvironments()->forAll(function ($key, RuntimeEnvironment $env) {
+          "upToDate" => $this->runtimeEnvironments->count() === $this->getExercise()->getRuntimeEnvironments()->count()
```

Comment:
Well... question is, if synchronization should be advertised on change of environment which is disabled in assignment. This is rather unimportant for the assignment author. But not much of a big problem...

Reply:
Synchronization should not be advertised if the changes are in a disabled environment only. But I agree that it is not big issue.

Reply:
If it's not a big issue, I would probably leave this as is for now. I feel like rephrasing the condition for enabled environments only won't be that easy.

Reply:
True would not be... Agree to postpone this